elliottlib/schema_group.yml

```diff
@@ -4,6 +4,19 @@
 type: map
 
 mapping:
+  # ID of the advisory we would auto attach bugs/builds to for the
+  # selected --group
+  "advisories":
```

Comment:
Actually, it occurs to me... Wouldn't this be better off in erratatool.yml? More topical and I don't think doozer needs it.

Reply:
Retracted; that's not what `erratatool.yml` is for. It's fine in `group.yml`.